src/App.jsx

```diff
@@ -1,26 +1,14 @@
-import { useEffect, useState } from 'react'
 import './App.css'
 import Layout from './views/Layout/Layout'
 import Home from './views/Home/Home'
-import { fetchUser } from './services/user'
+import { useUser } from './context/UserContext'
 
 function App() {
   // inital value should match the data type of end value
-  const [user, setUser] = useState({})
-
-  useEffect(() => {
-    fetchUser()
-      .then((fetchedUser) => {
-        setUser(fetchedUser)
-      })
-      .catch((error) => {
-        throw new Error(`Error: ${error}`)
-      })
-  }, [])
-
+  const { user } = useUser()
```

Comment:
Since you aren't using `user` in `App.jsx` anymore, this call to the `useUser` hook isn't necessary here.